src/App.jsx

```diff
@@ -1,38 +1,54 @@
-import React from 'react';
-import { Redirect, Route } from 'react-router-dom';
-import { IonReactRouter } from '@ionic/react-router';
-import { IonApp, IonHeader } from '@ionic/react';
-import Menu from './components/Menu';
-import Footer from './components/Footer';
-import ArticleList from './components/ArticleList';
+import React from 'react'
+import { connect } from 'react-redux'
+// import { Redirect, Route } from 'react-router-dom';
+// import { IonReactRouter } from '@ionic/react-router';
+import { IonApp, IonHeader } from '@ionic/react'
```

Comment:
I would remove these if they are not currently being used

Reply:
Done!